### ISSUE TITLE
Show TaskRun results when clicking on a TaskRun

### ISSUE THREAD

**Issue by fiunchinho** [tektoncd/dashboard#1943]
When clicking on a TaskRun I can see two columns: parameters and status. I think it'd be useful to add a third column to show the Tekton `results` (if any) that the `TaskRun` created for other tasks to consume.

**Comment by AlanGreene** [comment]
Thanks for the feature request @fiunchinho, I agree this would be useful. It falls under the 'currency' section of our roadmap to ensure the Dashboard surfaces relevant information about all Tekton Pipelines features to the user. I don't see an existing issue tracking this so thanks again for opening one.

**Comment by afrittoli** [comment]
Same for pipeline runs please 🙏 
The release pipeline now produces results with links to the release:
```
Name:              pipeline-release-nightly-mdb7d
Namespace:         tekton-nightly
Pipeline Ref:      pipeline-pipeline-release
Service Account:   default
Timeout:           1h0m0s
Labels:
 tekton.dev/pipeline=pipeline-pipeline-release
 triggers.tekton.dev/eventlistener=tekton-nightly
 triggers.tekton.dev/trigger=pipeline-nightly-release-nightly
 triggers.tekton.dev/triggers-eventid=krlxj

🌡️  Status

STARTED        DURATION     STATUS
10 hours ago   31 minutes   Succeeded

📦 Resources

 No resources

⚓ Params

 NAME                   VALUE
 ∙ package              github.com/tektoncd/pipeline
 ∙ gitRevision          ca26ea83aba58aa0313f1a5bb81fbd686a6acb01
 ∙ imageRegistry        gcr.io
 ∙ imageRegistryPath    tekton-nightly
 ∙ versionTag           v20210303-ca26ea83ab
 ∙ serviceAccountPath   release.json

📝 Results

 NAME                    VALUE
 ∙ commit-sha            ca26ea83aba58aa0313f1a5bb81fbd686a6acb01
 ∙ release-file          https://storage.googleapis.com/tekton-releases-nightly/pipeline/previous/v20210303-ca26ea83ab/release.yaml
 ∙ release-file-no-tag   https://storage.googleapis.com/tekton-releases-nightly/pipeline/previous/v20210303-ca26ea83ab/release.notag.yaml

📂 Workspaces

 NAME               SUB PATH   WORKSPACE BINDING
 ∙ workarea         ---        VolumeClaimTemplate
 ∙ release-secret   ---        Secret (secret=release-secret)

🗂  Taskruns

 NAME                                                              TASK NAME                  STARTED        DURATION     STATUS
 ∙ pipeline-release-nightly-mdb7d-report-bucket-qs8fh              report-bucket              10 hours ago   7 seconds    Succeeded
 ∙ pipeline-release-nightly-mdb7d-publish-to-bucket-latest-7bhmx   publish-to-bucket-latest   10 hours ago   25 seconds   Succeeded
 ∙ pipeline-release-nightly-mdb7d-publish-to-bucket-tvc2d          publish-to-bucket          10 hours ago   26 seconds   Succeeded
 ∙ pipeline-release-nightly-mdb7d-publish-images-4plv5             publish-images             10 hours ago   20 minutes   Succeeded
 ∙ pipeline-release-nightly-mdb7d-build-base-image-kfpmn           build-base-image           10 hours ago   1 minute     Succeeded
 ∙ pipeline-release-nightly-mdb7d-unit-tests-s6djm                 unit-tests                 10 hours ago   6 minutes    Succeeded
 ∙ pipeline-release-nightly-mdb7d-build-wrxcz                      build                      10 hours ago   4 minutes    Succeeded
 ∙ pipeline-release-nightly-mdb7d-precheck-sdxs8                   precheck                   10 hours ago   25 seconds   Succeeded
 ∙ pipeline-release-nightly-mdb7d-git-clone-t8xl7                  git-clone                  10 hours ago   33 seconds   Succeeded
```

But the results are not visible via the dashboard.

**Comment by AlanGreene** [comment]
PR open to display results for TaskRuns: https://github.com/tektoncd/dashboard/pull/1965
We'll include this in the v0.15 release which should be available next week.

Doing the same at the PipelineRun level will take a little more work as we don't have equivalent tabs *yet*... Maybe something we can try to tackle for v0.16. See https://github.com/tektoncd/dashboard/issues/1570 and https://github.com/tektoncd/dashboard/issues/762 for related issues and some discussions of the proposed design.

**Comment by AlanGreene** [comment]
TaskRun results have been included in the nightly builds for a few days now and will be included in the Dashboard v0.15 release later this week.

@afrittoli I'm moving the remaining work for PipelineRun results to #1570 as both require the same foundation.
